src/Mono.Android/Android.Net.Wifi.P2p/WifiP2pManager.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using Android.Runtime;
+
+namespace Android.Net.Wifi.P2p
+{
+	// This was converted to an enum in .NET 9
```

Comment:
So if the enum what people are supposed to use now? If so do we need an Obsolete attrbiute on these?

Reply:
Because this was not properly enumified when it was released, I assume there are methods that still require the constants instead of the enum, so I don't think we should obsolete them.